package.json

```diff
@@ -75,8 +75,8 @@
   },
   "scripts": {
     "build": "gatsby build",
-    "check-all": "yarn prettier && yarn lint && yarn flow",
-    "ci-check": "yarn prettier:diff && yarn lint && yarn flow",
+    "check-all": "npm-run-all prettier --parallel lint flow",
```

Comment:
Hm. Did this command work for you locally?

I needed to add a `flow` script entry before it would work for me.

```
$ yarn check-all
...
ERROR: Task not found: "flow"

$ npm run ci-check
...
ERROR: Task not found: "flow"
```

Fixed by:
```diff
diff --git a/package.json b/package.json
index 94e16032b..7ebc70ae4 100644
--- a/package.json
+++ b/package.json
@@ -78,6 +78,7 @@
     "check-all": "npm-run-all prettier --parallel lint flow",
     "ci-check": "npm-run-all prettier:diff --parallel lint flow",
     "dev": "gatsby develop -H 0.0.0.0",
+    "flow": "flow",
     "lint": "eslint .",
     "netlify": "yarn install && yarn build",
     "prettier": "prettier --config .prettierrc --write \"{gatsby-*.js,{flow-typed,plugins,src}/**/*.js}\"",
```

Reply:
I'm just going to add this myself 😁